### ISSUE TITLE
Resource problem..

### ISSUE THREAD

**Issue by AybimYazilim** [skipperbent/simple-php-router#224]
SimpleRouter::resource('/m1', 'KurumsalController')->name('kurumsal'); // Kurumsal
        SimpleRouter::resource('/m2', 'HizmetController')->name('hizmet'); // Hizmetler
        SimpleRouter::resource('/m3', 'ProjeController')->name('proje'); // Projeler
        SimpleRouter::resource('/m4', 'ReferansController')->name('referans'); // Projeler
        SimpleRouter::resource('/m5', 'BlogController')->name('blog'); // Blog
        SimpleRouter::resource('/m6', 'VideoController')->name('video'); // Video
        SimpleRouter::resource('/m7', 'FotoController')->name('foto'); // Foto Galeri
        SimpleRouter::resource('/m11', 'DoktorController')->name('doktor'); // Doktorlar

when i try to enter "x.com/m11", its running "KurumsalController" which is "m1" 

how can i fix it

Thanks

**Comment by skipperbent** [comment]
I could imagine that `/m1` matches `/m11` before it has the chance to check `/m11`.

Until this bug is fixed, I think you can solve it for now by moving 

`SimpleRouter::resource('/m11', 'DoktorController')->name('doktor'); // Doktorlar`

before this route

`SimpleRouter::resource('/m1', 'KurumsalController')->name('kurumsal'); // Kurumsal`

Like this:

```php
    SimpleRouter::resource('/m11', 'DoktorController')->name('doktor'); // Doktorlar
    SimpleRouter::resource('/m1', 'KurumsalController')->name('kurumsal'); // Kurumsal
    SimpleRouter::resource('/m2', 'HizmetController')->name('hizmet'); // Hizmetler
    SimpleRouter::resource('/m3', 'ProjeController')->name('proje'); // Projeler
    SimpleRouter::resource('/m4', 'ReferansController')->name('referans'); // Projeler
    SimpleRouter::resource('/m5', 'BlogController')->name('blog'); // Blog
    SimpleRouter::resource('/m6', 'VideoController')->name('video'); // Video
    SimpleRouter::resource('/m7', 'FotoController')->name('foto'); // Foto Galeri
```

**Comment by skipperbent** [comment]
@AybimYazilim I can't simulate the same error on the latest version (3.x). Which version are you using?

**Comment by AybimYazilim** [comment]
I am using v3.1 but i still see this error.

This bug fixed moving this

SimpleRouter::resource('/m11', 'DoktorController')->name('doktor'); // Doktorlar

before this route

SimpleRouter::resource('/m1', 'KurumsalController')->name('kurumsal'); // Kurumsal

**Comment by skipperbent** [comment]
@AybimYazilim 

Strange.. I'll have to look into this bug in debt - but until it's fixed, keep an eye for urls with similar paths if you experience any similar errors.

I will keep this issue open until the issue has been resolved.

Thanks for your feedback!

**Comment by AybimYazilim** [comment]
Thanks for your support

**Comment by skipperbent** [comment]
This bug has been resolved in the latest release 3.2.0.

Please ensure to read the release notes before updating, available here:
https://github.com/skipperbent/simple-php-router/releases
